Hackney.Shared.HousingSearch.Tests/Factories/DomainFactoryTests.cs

```diff
@@ -44,5 +46,46 @@ public void CanConvertQueryableTempAccommodationOfficerEntityToDomain()
             domainTAOfficer.FirstName.Should().Be(queryableTAOfficer.FirstName);
             domainTAOfficer.LastName.Should().Be(queryableTAOfficer.LastName);
         }
+
+        [Fact]
+        public void CanConvertQueryableAssetContractEntityToDomain()
+        {
+            var queryableAssetContract = _fixture.Create<QueryableAssetContract>();
+
+            var domainContract = queryableAssetContract.ToDomain();
+
+            domainContract.Id.Should().Be(queryableAssetContract.Id);
```

Comment:
Would be good to cover all properties in the target object to ensure the conversion is correct for entire object.

Reply:
I think `TargetType` is still missing?